src/token/types.ts

```diff
@@ -56,7 +56,7 @@ export type GetTokensOptions = {
  * Note: exported as public Type
  */
 export type Token = {
-  address: Address; // The address of the token contract
+  address: Address | ''; // The address of the token contract
```

Comment:
this is since ETH has `''` as the `Address` field

Reply:
is this always going to be true?

Reply:
When we say ETH, are we talking about ETH Base or ETH (L1)?

Reply:
either, native ETH does not have an address on any network

also this PR isn't ready yet, will be doing some refactoring once i make some upstream changes to the api

Reply:
awesome